Sources/Kitura/staticFileServer/StaticFileServer.swift

```diff
@@ -101,9 +102,25 @@ open class StaticFileServer: RouterMiddleware {
     /// - Parameter customResponseHeadersSetter: an object of a class that
     /// implements `ResponseHeadersSetter` protocol providing a custom method to set
     /// the headers of the response.
-    public init(path: String = "./public", options: Options = Options(),
+    public init?(path: String = "./public", options: Options = Options(),
```

Comment:
This is a breaking change.  For Kitura 2.x, we'll need to leave it as a non-failable initializer, and log at `.error` instead.